cypress/support/cypress-support/src/utils.js

```diff
@@ -1001,20 +1001,46 @@ global.resolveAtRuntime = function (input) {
           [pattern, functionName] = pattern.split('.');
         }
 
-        // If a function name is present in the pattern, call the function with pattern content as input.
-        // Reading the pattern content from the runtime environment variable
-        const value = runtimeEnv[pattern] !== undefined ? runtimeEnv[pattern] : match;
-        return functionName && functions[functionName] ? functions[functionName](value) : value;
+        // If the input contains '->', Consdering it as an object path and extracting the value from the object
+        if (pattern.includes('->')) {
+          return extractValueFromObjectPath(pattern);
+        } else {
```

Comment:
Here I think instead of an 'else' we can just check if there is a functionName and it should handle the case I mentioned in the call this morning.

Reply:
addressed the comment
First we are going to fetch the value from the object then calling function to change the case